dolphinscheduler-ui-next/src/views/resource/task-group/queue/use-table.ts

```diff
@@ -149,14 +148,8 @@ export function useTable(
           }
 
           item.taskGroupName = taskGroupName
-          item.createTime = format(
-            parseTime(item.createTime),
-            'yyyy-MM-dd HH:mm:ss'
-          )
-          item.updateTime = format(
-            parseTime(item.updateTime),
-            'yyyy-MM-dd HH:mm:ss'
-          )
+          item.createTime = renderFormat(item.createTime)
```

Comment:
CreateTime doesn't need to be formatted here, it will be formatted when the table is rendered.